src/icrar/leap-accelerate/ms/MeasurementSet.h

```diff
@@ -53,13 +53,14 @@ namespace icrar
         std::unique_ptr<casacore::MSColumns> m_msc;
 
         boost::optional<std::string> m_filepath;
+        std::set<std::int32_t> m_antennas;
```

Comment:
This is used only locally within `MeasurementSet::MeasurementSet` so maybe instead of being a member it could simply be a local variable in the ctor.

Reply:
I think there may be some more validation I can do with this such as checking the that the missing indices in this set should be in the flagged set. It's currently not used for much but since it's calculated anyway there's only a slight peek memory overhead in keeping it as a member